src/Providers/Stripe/Payment/Service.php

```diff
@@ -43,22 +45,16 @@ public function create(
         Token $token,
         int $amount,
         string $currency = 'GBP',
-        string $description = null,
-        array $metadata = []
+        array $metadata = [],
+        $customer = null,
+        string $description = null
     ): Payment {
         switch ($token->type) {
             case StripeToken::CUSTOMER:
-                $paymentMethods = PaymentMethod::all([
-                    'customer' => $token->value,
-                    'type' => 'card',
-                ]);
-
-                /** @var PaymentMethod $paymentMethod */
-                $paymentMethod = $paymentMethods->data[0];
+                $cards = $this->cards()->index($token);
+                $card = $cards[0];
```

Comment:
why has this changed?

Reply:
Because we have a function of the CardService that is doing exactly that. So I thought it was best to use it instead.